requirements/dev.txt

```diff
@@ -82,7 +82,7 @@ pytest==7.1.3
     #   pytest-pep8
 pytest-cache==1.0
     # via pytest-pep8
-pytest-cov==3.0.0
+pytest-cov==4.0.0
```

Comment:
dropped support for multi-processing https://pytest-cov.readthedocs.io/en/latest/changelog.html#id1